### ISSUE TITLE
Local building for `lineage-20.0` is failing

### ISSUE THREAD

**Issue by petefoth** [lineageos4microg/docker-lineage-cicd#441]
What am I doing wrong for `lineage-20.0`?
```
docker run \
    -e BRANCH_NAME="lineage-20.0" \
    -e DEVICE_LIST="payton" \
    -v "~/android/build/lineage:/srv/src" \
    -v "~/android/build/zips:/srv/zips" \
    -v "~/android/build/logs:/srv/logs" \
    -v "~/android/build/cache:/srv/ccache" \
    -v "~/android/build/manifests:/srv/local_manifests" \
    lineageos4microg/docker-lineage-cicd

Set cache size limit to 50.0 GB
>> [Sun May  7 23:46:45 UTC 2023] Building branch lineage-20.0 is not (yet) suppported
```

_Originally posted by @js290 in https://github.com/lineageos4microg/docker-lineage-cicd/issues/335#issuecomment-1537570904_

**Comment by petefoth** [comment]
At the moment, for local  `lineageos-20.0` builds to work, they need to use the `latest` or `master` tags. The build server seems to do that, even though the tag isn't present in the `docker run ...` command. 

Local builds need to specify the tag, otherwise they pick up out-of-date builds. For example, I have the following images on my machine. If I call `docker run...` with no tag for` lineageos-20.0` branch it runs the latest `<none>` image `85e4b980063f`, which fails with the `Building branch lineage-20.0 is not (yet) suppported` error

```
$ docker images
REPOSITORY                             TAG          IMAGE ID       CREATED        SIZE
lineageos4microg/docker-lineage-cicd   latest       fb3dd536126a   8 weeks ago    2.05GB
lineageos4microg/docker-lineage-cicd   master       fb3dd536126a   8 weeks ago    2.05GB
lineageos4microg/docker-lineage-cicd   lineage-20   60aa571d62d4   6 months ago   2.06GB
lineageos4microg/docker-lineage-cicd   <none>       85e4b980063f   9 months ago   2.05GB
lineageos4microg/docker-lineage-cicd   <none>       e7b5f8b26633   9 months ago   2.05GB
```

We need to find out ***why*** this works on the server, but not for local builds, and update the README if necessary

**Comment by petefoth** [comment]
The build script on the server starts with a call to `docker pull lineageos4microg/docker-lineage-cicd` which will pull the latest version from DockerHub

```
  Using default tag: latest
  latest: Pulling from lineageos4microg/docker-lineage-cicd
  Digest: sha256:4172b3378ea386c9bfb256687ca08503b6d853b96643bedd2588555b00f95098
  Status: Image is up to date for lineageos4microg/docker-lineage-cicd:latest
```
I have updated the README.md and the change will be merged in PR #445

**Comment by petefoth** [comment]
Fixed in https://github.com/lineageos4microg/docker-lineage-cicd/pull/484/commits/84765a978400915857104d5123788053c98ccd78

To merge in 484
